### PR TITLE
[wasm] Remove unnecessary console messages.

### LINKED ISSUE #40721: Blazor WebAssembly Debugging
Setting a tracepoint in Visual Studio and running a Blazor WebAssembly App every time a tracepoint is hit there is additional information written to the console:

```
ss_reqs 0 bp_reqs 1
mono_wasm_fire_bp
```

This pollutes the output and makes it hard to debug. 

Is this something the runtime controls or should the issue be raised via VS Feedback?

### PR DESCRIPTION
**Pull request by thaystg**
Changing some console.log to console.debug to not pollute the console.
Changing printf to DEBUG_PRINTF to follow the debug level chosen by user.
Remove unnecessary console.log.
Fix #40721

### CODE REVIEW

**Review by CoffeeFlux — APPROVED** [review]

**Review by vargaz — APPROVED** [review]

**Review by lewing — APPROVED** [review]

File: src/mono/wasm/runtime/library_mono.js [hunk 1]
```diff
@@ -902,7 +902,7 @@ var MonoSupportLib = {
 			if (ctx.tracing)
 				console.log ("MONO_WASM: Loaded:", asset.name, "size", bytes.length, "from", url);
 			else
-				console.log ("MONO_WASM: Loaded:", asset.name);
+				console.debug ("MONO_WASM: Loaded:", asset.name);
```
Comment (lewing) [inline src/mono/wasm/runtime/library_mono.js:905]: I'm not sure we need this at all

Comment (thaystg) [reply src/mono/wasm/runtime/library_mono.js]: Removed.

**Review by radical — APPROVED** [review]

**Comment by lewing** [comment]
Strange failure here in the file system layer @steveisok @mdh1418.  Doesn't look related to this change.

**Review by lewing — APPROVED** [review]

**Comment by lewing** [comment]
Failure isn't change related

**Comment by lewing** [comment]
@steveisok let me know if you aren't tracking the FS related failure.

**Pull request merged**